pkg/catalog/diagram.go

```diff
@@ -120,7 +169,7 @@ func (p *Generator) CreateSequenceDiagram(appName string, endpoint *sysl.Endpoin
 		os.Exit(1)
 		return ""
 	}
-	return p.CreateFile(plantumlString, plantuml, appName, endpoint.GetName()+p.Ext)
+	return p.CreateFile(plantumlString, plantuml, appName, endpoint.GetName()+"_seq_"+p.Ext)
```

Comment:
was there a problem with `_sequence_`?

Reply:
Yeah, the files seem to clash when "_seq_" wasn't appended. Was `_sequence_` used in the past or do you think it's a more suitable suffix?